src/arkit.cs

```diff
@@ -1245,11 +1246,15 @@ interface ARReferenceObject : NSSecureCoding {
 		NSSet<ARReferenceObject> GetReferenceObjects (string resourceGroupName, [NullAllowed] NSBundle bundle);
 
 		[Export ("exportObjectToURL:previewImage:error:")]
-		bool ExportObject (NSUrl url, [NullAllowed] UIImage previewImage, [NullAllowed] out NSError error);
+		bool Export (NSUrl url, [NullAllowed] UIImage previewImage, [NullAllowed] out NSError error);
 
 		[Export ("referenceObjectByApplyingTransform:")]
 		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
-		ARReferenceObject GetReferenceObject (Matrix4 transform);
+		ARReferenceObject ApplyingTransform (Matrix4 transform);
```

Comment:
much better, but I think `ApplyTransform` is more .nutty :)

Reply:
Yeah I was hesitating with that and couldn't really make up my mind (: